tools/src/wyvern/tools/tests/EffectSystemTests.java

```diff
@@ -384,9 +384,85 @@ public void testNetwork19() throws ParseException {
         TestUtil.doTestScriptModularly(PATH, "effects.testNetwork19", Util.stringType(), new StringLiteral("Network19 with effects"));
     }
 
+    @Test
+    public void accepted2() throws ParseException {
+        TestUtil.doTestScriptModularly(PATH, "higherOrderEffects.acceptedClient2", Util.stringType(), new StringLiteral("abc"));
+    }
+
     @Test
     public void testObjNetwork() throws ParseException {
         /* Object with effect annotations. */
         TestUtil.doTestScriptModularly(PATH, "effects.objNetwork", Util.stringType(), new StringLiteral("ObjNetwork with effects"));
     }
+
+    @Test
+    public void testSeparation1() throws ParseException {
```

Comment:
Why is this PR and the test suit called Separation? Please add a description for this PR and include the answer to this question.